docs/changelog/2023-09-29.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Sep 29, 2023
+tags: []
+---
+
+## Overview
+
+### Features & Improvements
+
+1. Visitors will now be able to carry on conversations over email
+2. Widget summary page!
+3. Improve website scraping robustness
+4. Fixed random name / colour generator for new visitors occasionally selecting an invalid value
+5. Improved billing report accuracy
+6. Fixed bug where changing languages deletes all suggested questions from widgets
+7. Fixed Stripe bug involving checkouts with a trial
+   8 Better escalation notifications — we make sure the visitor details are in the notifications
+
+### Coming soon...
+
+1. Tools - Customer details collection + email notifications
```

Comment:
I think we can remove tools/auth visitors from this list (we mentioned them last week).